public/js/pimcore/asset/image.js

```diff
@@ -427,30 +445,48 @@ pimcore.asset.image = Class.create(pimcore.asset.asset, {
 
         var markerDD = new Ext.dd.DD(marker);
 
-        this.marker = marker;
-    },
+        Ext.getCmp('remove_focal_point').setVisible(true);
+        Ext.getCmp('add_focal_point').setVisible(false);
 
-    getSaveData : function ($super, only) {
-        var parameters = $super(only);
+        this.marker = marker;
 
+    },
+    removeFocalPoint: function () {
```

Comment:
in `addFocalPoint`,  a context menu on the focal point element (line 425) is added that has the logic to delete a focal point, you should maybe replace the anonymous handler with this method.

Reply:
Thank you!
https://github.com/pimcore/admin-ui-classic-bundle/pull/331/commits/15685e1ed732d09dc5bbeb3ff2b017b4e587680c